duo_client/accounts.py

```diff
@@ -15,6 +17,12 @@ def get_child_accounts(self):
         response = self.json_api_call('POST',
                                       '/accounts/v1/account/list',
                                       params)
+        if response and isinstance(response, list):
+            for account in response:
+                account_id = account.get('account_id', None)
+                api_hostname = account.get('api_hostname', None)
+                if account_id and api_hostname:
+                    Accounts.child_map[account_id] = api_hostname
```

Comment:
How likely is it that an account_id's hostname will change?  i.e. any realistic chance this info will be outdated within the expected lifetime of the map?

Reply:
There is a 0% chance the account_id's hostname will change.  Even in a migration scenario, this information is static.

Reply:
great news